### DISCUSSION TITLE
Developer Debugging Features

### DISCUSSION THREAD

**Discussion by eliotwrobson** [PrairieLearn/PrairieLearn#9467] [Ideas]
Creating a discussion to collect ideas on features intended to improve quality of life for question developers. There have been a few issues that describe features that fall into this category:

- https://github.com/PrairieLearn/PrairieLearn/issues/2272
- https://github.com/PrairieLearn/PrairieLearn/issues/7337
- https://github.com/PrairieLearn/PrairieLearn/issues/2279 (and https://github.com/PrairieLearn/PrairieLearn/pull/7983)
- https://github.com/PrairieLearn/PrairieLearn/issues/8859
- https://github.com/PrairieLearn/PrairieLearn/issues/5896
- https://github.com/PrairieLearn/PrairieLearn/issues/2273
- Maybe https://github.com/PrairieLearn/PrairieLearn/issues/1502
- https://github.com/PrairieLearn/PrairieLearn/issues/8640

**Comment by mwest1066** [comment]
Oh yes, it would be so great to improve this!

**Reply by eliotwrobson** [reply]
I think #7337 gives the ability to expand on a lot of these. Many of these aren't suitable to enable as normal issues but still useful information for developers
